ngx-tools/testing/src/mocks/retry-with-escalation.mock.ts

```diff
@@ -27,6 +26,9 @@ export const SCHEDULER = new InjectionToken<Scheduler>('scheduler');
 export const ESCALATION_WAIT_TIME = new InjectionToken<number>('wait time');
```

Comment:
Some items definitely had to move around to remove some duplicate code. Not sure if that's what you're referring to..

Reply:
Ah probably what you're referring to is the changes needed for the move to the JWT testing dir. Those changes weren't needed here so I left it all as it currently is since it's deprecated.